src/mega-plusplus.coffee

```diff
@@ -43,6 +43,8 @@ module.exports = (robot) ->
   reasonsKeyword = process.env.HUBOT_PLUSPLUS_REASONS or 'raisins'
   maxPoints = process.env.HUBOT_PLUSPLUS_MAX_POINTS or 5
   reasonConjunctions = process.env.HUBOT_PLUSPLUS_CONJUNCTIONS or 'for|because|cause|cuz|as'
+  selfKarmaScolding = process.env.SELF_KARMA_SCOLDING or 'Did you really just try to give yourself karma?  REALLY?!'
+  humilityScolding = process.env.HUMILITY_SCOLDING or 'There there, buddy... it\'s gonna be all right.'
```

Comment:
lets make these names consistent - keep `selfKarmaScold` for self++ and `selfKarmaSupport`